src/panels/config/info/integrations-card.ts

```diff
@@ -56,7 +60,9 @@ class IntegrationsCard extends LitElement {
                       />
                     </td>
                     <td class="name">
-                      ${domainToName(this.hass.localize, domain)}<br />
+                      ${manifest
+                        ? integrationName(this.hass.localize, manifest, domain)
+                        : domainToName(this.hass.localize, domain)}<br />
```

Comment:
If we add manifest as optional param, we can just do:
```suggestion
                      ${domainToName(this.hass.localize, domain, manifest)}<br />
```

Reply:
Adjusted 👍